src/App.js

```diff
@@ -43,17 +43,65 @@ class App extends Component {
   }
 
   findSearchedTeam = (input) => {
+    if (input === null || input === undefined) {
+      return;
+    }
+
     let matched = nflTeams.filter(team => {
       return team.name.toLowerCase().includes(input.toLowerCase())
     })
+
     this.setState({nflTeams: matched})
  }
+  
+  findSelectedDivision = (input) => { 
+    let matched; 
+    if (input.length === 3) {
+      matched = nflTeams.filter(team => {
+
+        return team.division.substring(0, 3).toLocaleLowerCase() === input;
+      })
+      this.setState({nflTeams: matched})
+    } else {
+
+      var alterInput = input.split('-');
+      alterInput[0] = alterInput[0].toUpperCase();
+
+      alterInput[1] = alterInput[1].charAt(0).toUpperCase() + alterInput[1].substring(1)
+
+      input = alterInput.join(' ');
+
+      matched = nflTeams.filter(team => {
+        return team.division === input;
+      })
+      this.setState({nflTeams: matched})
+    }
+  }
 
   render() {
     if (this.state.isLoading) {
       return(
         <div className="loading-screen"> <img className="loading-image" src={loadingGif} alt="loading screen"/> </div>
         )
+    } else if (this.state.nflTeams.length === 0) { 
+      return (
+        <div className="app">
+          <div className='app-title-area'>
+            <h1 className='app-title'>
+              <img src={nflLogo} alt="" />
+              The Traveling Fan
+              <img src={nflLogo} alt="" />
+            </h1>
+          </div>
+          <div>
+            <Nav findSearchedTeam={this.findSearchedTeam} findSelectedDivision={this.findSelectedDivision} />
+          </div>
+          <div className="flex">
+            <h1>PLEASE TRY AGAIN</h1>
+            {/* <Container nflTeams={this.state.nflTeams} cities={this.state.cities} /> */}
```

Comment:
lets expand upon this in the future but I like an error message when your search doesn't return anything!